src/python/pants/engine/internals/parser.py

```diff
@@ -1,4 +1,4 @@
-# Copyright 2015 Pants project contributors (see CONTRIBUTORS.md).
+# Copyright 2020 Pants project contributors (see CONTRIBUTORS.md).
```

Comment:
why?

Reply:
The file has diverged a lot from 2015, particularly from my changes in the past month. Generally, Stu and I have been updating the year for a file when it's seen major changes, such as changing `graph.py`.

I'll revert though - not a big deal either way.